Dockerfile

```diff
@@ -1,33 +1,34 @@
-ARG DOCKER_COMPOSE=docker/compose:1.24.1
-ARG DOCKER=docker:19.03.1
-ARG BUILDKIT=moby/buildkit:v0.6.1
-ARG GOLANG=golang:1.13.6-alpine
+ARG DOCKER_COMPOSE=docker/compose:1.26.2
+ARG DOCKER=docker:19.03.12
+ARG BUILDKIT=moby/buildkit:v0.7.1
+ARG GOLANG=golang:1.14-alpine
 
 FROM $DOCKER_COMPOSE as docker_compose
 FROM $DOCKER as docker
 FROM $BUILDKIT as buildkit
 
 FROM $GOLANG as tc-redirect-tap
 
-ENV FIRECRACKER_SDK_VERSION=v0.19.0
+ARG FIRECRACKER_SDK_VERSION=v0.19.0
 
 WORKDIR /src
 
 RUN apk add --update --no-cache \
   curl \
   make
 
+# TODO: switch to awslabs/tc-redirect-tap once it has a release
 RUN curl -LO https://github.com/firecracker-microvm/firecracker-go-sdk/archive/${FIRECRACKER_SDK_VERSION}.tar.gz
 RUN tar xzf ${FIRECRACKER_SDK_VERSION}.tar.gz --strip-components=1
 RUN make -C cni install
 
-FROM alpine:3.9
+FROM alpine:3.11
 
-ENV BUILDX_VERSION=v0.3.0
-ENV GITMETA_VERSION=v0.1.0-alpha.3
-ENV CLOUD_SDK_VERSION=258.0.0
-ENV CNI_PLUGINS_VERSION=v0.8.5
-ENV FIRECRACKER_VERSION=v0.21.0
+ARG GITMETA_VERSION=v0.1.0-alpha.3
```

Comment:
I don't think we depend on gitmeta at all anymore.

Reply:
do we use `docker-compose` anywhere?

Reply:
I don't think we do.

Reply:
removed those two to make it a bit more sllim